### PR TITLE
FOUR-15487:Phase1: Remove Analytics chart from future and existing customers in dev  (QuickSight related)

### PR DESCRIPTION
**Pull request by fagubla**
## Issue & Reproduction Steps
Remove Analytics chart from future and existing customers in dev (QuickSight related)
Expected behavior: 

Actual behavior: 

## Solution
-

## How to Test
Test the steps above

## Related Tickets & Packages
- [FOUR-15487](https://processmaker.atlassian.net/browse/FOUR-15487)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next
ci:package-dynamic-ui:feature/FOUR-15487

[FOUR-15487]: https://processmaker.atlassian.net/browse/FOUR-15487?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-9a8ef20acc.engk8s.processmaker.net

**Review by luNunezProcessmaker — APPROVED** [review]

**Review by ana-mauricio — CHANGES_REQUESTED** [review]
The following observations were found :
1. The sizes of the cards are different (ANALYTICS CHART height is higher)
image
2. The Welcome user is above the avatar
image
3. The image in Anakytics chart is not responsive  
https://github.com/ProcessMaker/screen-builder/assets/21198450/c6d3ad88-a185-46a9-b369-b184886dfd94


@fagubla

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-9a8ef20acc.engk8s.processmaker.net

**Review by ana-mauricio — APPROVED** [review]
The WELCOME was corrected, and the others observations works as expected behavior 
@fagubla

File: src/components/renderer/form-analytics-chart.vue [hunk 1]
```diff
@@ -41,10 +43,14 @@ export default {
...
       this.graphic = this.listChartOption;
     }
+    this.verifyChart();
   },
   methods: {
     openExternalLink() {
       window.open("/package-analytics-reporting", "_blank");
+    },
+    verifyChart() {
+      this.showChart = this.graphic.name === "PM Analytics Chart" || this.graphic.name === "PM Analytics Dashboard";
```
Comment (pmPaulis) [inline src/components/renderer/form-analytics-chart.vue:53]: Please this default needs to show when the Analytics does not have any chart selected like: 
image

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-9a8ef20acc.engk8s.processmaker.net

**Review by luNunezProcessmaker — APPROVED** [review]

**Review by luNunezProcessmaker — APPROVED** [review]

**Pull request merged**